CHANGELOG

```diff
@@ -15,6 +15,8 @@ devel
 * Added logging of elapsed time of ArangoSearch commit/consolidation/cleanup 
   jobs.
 
+* added list-repeat AIR primitive.
```

Comment:
```suggestion
* Added list-repeat AIR primitive.
```

Reply:
Maybe write add a bit more info here what list-repeat actually does.

Reply:
Sth. like: e.g. For use in ArangoDB Custom Pregel in order to ...

Reply:
```suggestion
* Added list-repeat AIR primitive that creates a list containing n copies of the input value.
```